samples/drivers/dac/boards/ek_ra2a1.overlay

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2025 Renesas Electronics Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	zephyr,user {
+		dac = <&dac0>;
+		dac-channel-id = <0>;
+		dac-resolution = <12>;
+	};
+};
+
+&dac {
+	dac0: dac@0 {
+		status = "okay";
+	};
+};
```

Comment:
This was already done on board-level, so we don't need to repeat it here. Same for all other overlays in this sample.

```suggestion
```